streams/src/test/java/org/apache/kafka/streams/processor/internals/AbstractTaskTest.java

```diff
@@ -96,6 +97,8 @@ public void shouldThrowWakeupExceptionOnInitializeOffsetsWhenWakeupException() {
     public void shouldThrowLockExceptionIfFailedToLockStateDirectoryWhenTopologyHasStores() throws IOException {
         final Consumer consumer = EasyMock.createNiceMock(Consumer.class);
         final StateStore store = EasyMock.createNiceMock(StateStore.class);
+        expect(store.name()).andReturn("dummy-store-name").anyTimes();
```

Comment:
This is kind of funny. We always called `store.name()`, but with the current changes, we call it twice. Apparently, EasyMock returns `null` on the first invocation and throws an exception on the second, which doesn't seem quite right.

Anyway, since we need the store to have a name, I made one.